include/xtensor/xiterator.hpp

```diff
@@ -449,6 +458,31 @@ namespace xt
         m_it = p_c->data_xend(l);
     }
 
+    template <class C>
+    template <class R>
+    inline R xstepper<C>::step_simd()
+    {
+        R reg;
+        reg.load_unaligned(&(*m_it));
+        m_it += xsimd::simd_batch_traits<R>::size;
```

Comment:
`reverse_simd_traits` should be preferred to `simd_batch_traits` which should be reserved for `xsimd` implementation.